Index: src/main/java/com/qf/Git.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/qf/Git.java	(revision d5421a3f7f855954b62ee2b8fedcacbd489cff66)
+++ src/main/java/com/qf/Git.java	(date 1596543684906)
@@ -2,6 +2,10 @@
 
 public class Git {
     public static void main(String[] args) {
+
         System.out.println("啥玩意？？？？？？");
+
+        System.out.println("曙光是个大帅b");
+
     }
 }
